scene/gui/scroll_container.h

```diff
@@ -72,13 +72,21 @@ class ScrollContainer : public Container {
 
 	struct ThemeCache {
 		Ref<StyleBox> panel_style;
+#ifdef TOOLS_ENABLED
+		Ref<StyleBox> focus_style;
```

Comment:
I wasn't sure if we should do the same for the `focus_style` since it will be confusing if the users have found and used it for some reason then their game throw errors when it can't find it after exporting, so i think it's ok to leave  it as is without `TOOLS_ENABLED` check and to bind it without the check too..